cli/types/index.d.ts

```diff
@@ -2934,6 +2942,130 @@ declare namespace Cypress {
     (fn: (currentSubject: Subject) => void): Chainable<Subject>
   }
 
+  /**
+   * These events come from the application currently under test (your application). These are the most useful events for you to listen to.
+   * @see https://docs.cypress.io/api/events/catalog-of-events.html#App-Events
+   */
+  interface Actions {
+    /**
+     * Fires when an uncaught exception occurs in your application.
+     * Cypress will fail the test when this fires.
+     * Return `false` from this event and Cypress will not fail the test. Also useful for debugging purposes because the actual `error` instance is provided to you.
+     * @example
+     * // likely want to do this in a support file
+     * // so it's applied to all spec files
+     * // cypress/support/index.js
+     *
+     * Cypress.on('uncaught:exception', (err, runnable) => {
+     *   // returning false here prevents Cypress from
+     *   // failing the test
+     *   return false
+     * })
+     * @see https://docs.cypress.io/api/events/catalog-of-events.html#App-Events
+     */
+    (action: 'uncaught:exception', fn: (error: Error, runnable: Mocha.IRunnable) => false | void): void

```

Comment:
The docs said it is the mocha runnable. I'm assuming this is the interface the docs meant

Reply:
I included an example only in this action. This action at least gives an template for other actions to have examples added.